java/src/jmri/jmrix/dccpp/swing/mon/DCCppMonPane.java

```diff
@@ -15,31 +29,110 @@
  * @author Paul Bender Copyright (C) 2004-2014
  * @author Giorgio Terdina Copyright (C) 2007
  * @author Mark Underwood Copyright (C) 2015
+ * @author Costin Grigoras Copyright (C) 2019
  */
 public class DCCppMonPane extends jmri.jmrix.AbstractMonPane implements DCCppListener {
+    private static final long serialVersionUID = 1L;
```

Comment:
JMRI doesn't use Java serialization, so `serialVersionUID` definitions aren't needed.

Reply:
Ack, will disable the respective warning for this project. Thanks for the quick merge!